src/components/annotations/attributes.js

```diff
@@ -322,11 +323,7 @@ module.exports = templatedArray('annotation', {
         editType: 'calc',
         description: [
             'Sets the annotation\'s x coordinate axis.',
-            'If set to an x axis id (e.g. *x* or *x2*), the `x` position',
-            'refers to an x coordinate',
-            'If set to *paper*, the `x` position refers to the distance from',
-            'the left side of the plotting area in normalized coordinates',
-            'where 0 (1) corresponds to the left (right) side.'
+            axisPlaceableObjs.axisRefDescription('x', 'left', 'right'),
```

Comment:
In addition to `xref` could we use "domain" in `axref` as well?

Reply:
yes we can, but `xref` has to be exactly the same as `axref` or else `axref` reverts to `pixel`.

Reply:
Cool! Thanks for the info.
So let's update the `axref` and `ayref` as well.

Reply:
fixed by 759d37bf9e1eac56cef5d1e4df44e899e20ffadc

Reply:
(sorry wrong review item)